### DISCUSSION TITLE
How to make a Linux distributable of Bforartists

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#3012] [Releases]
From what I see, we have 3 distributables:
-- tar.xz
-- deb
-- AppImage

I am assuming the compiled build would be zipped up in a `tar.xz`
But how do you make a `*.deb` package?
Are the instructions here in #2569 [https://github.com/Bforartists/Bforartists/discussions/2569](https://github.com/Bforartists/Bforartists/discussions/2569) still relevant? 

Just digging for clues here.

**Comment by ReinerBforartists** [comment]
Please be patient. I will come to it ;)

There is a bit more involved for linux than just pressing the compile button in VS. I plan to make a tutorial for how to create the release binaries. Just two or three weeks to go until Blender 3.3 release ...
